application/templates/components/check-transform-base.html

```
@@ -0,0 +1,342 @@
+              </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+              {% for row in entities_data.rows %}
+              {% if row.category == 'changed' %}{% set _row_bg = '#ffd8b0' %}
+              {% elif row.category == 'in_both' %}{% set _row_bg = '#fff7bf' %}
+              {% elif row.category == 'new' %}{% set _row_bg = '#d4edda' %}
+              {% else %}{% set _row_bg = '#d0e4f7' %}{% endif %}
+              <tr class="govuk-table__row" style="background-color: {{ _row_bg }};">
+                {% for col in entities_data.columns %}
+                {% set _changed = row.changed_fields and col in row.changed_fields %}
+                <td class="govuk-table__cell app-wrap{% if _changed %} app-cell-changed{% endif %}"{% if _changed %} title="Platform value: {{ row.changed_fields[col] }}" data-platform-value="Platform value: {{ row.changed_fields[col] }}"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)